services/madoc-ts/src/frontend/shared/components/SearchResults.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import { SearchResult } from '../../../types/schemas/search';
+
+import { SearchBox } from '../atoms/SearchBox';
+import { GridContainer } from '../atoms/Grid';
+import { GridColumn, Dropdown } from '@capture-models/editor';
+import { Link } from 'react-router-dom';
+
+const ResultsContainer = styled.div`
+  min-width: 70vw;
```

Comment:
This was causing an issue for me on a wider display
![Screenshot 2020-09-09 at 19 28 05](https://user-images.githubusercontent.com/8266711/92638703-97ad6a80-f2d2-11ea-9c88-ab586634bace.png)

I think we could use `70%` or maybe a fixed pixel width on the left hand column and `flex: 1 1 0px` on the results container.